### ISSUE TITLE
Cache JFrog Repositories

### ISSUE THREAD

**Issue by JXavierMSFT** [Azure/acr#683]
Caching for ACR will soon allow users to cache repositories from JFrog. This functionality is scheduled for release in late June.

**Comment by SimonHarrisonSH** [comment]
Commenting, to get updates

**Comment by stacda01** [comment]
Commenting for updates

**Comment by mabenedi** [comment]
Hello, I'm on the team working on the feature. I'm not confident we'll have JFrog ready in July, but we are still actively working on it.

**Comment by zagaria** [comment]
Hello @mabenedi 

Does JFrog Repositories mean releases-docker.jfrog.io? Or it is about all JFrog registries (include on premise installation)?

**Comment by mabenedi** [comment]
We plan to eventually support both *.Jfrog.io upstreams and self hosted Jfrog. That support might not come at the same time though.

**Comment by dlorych** [comment]
Hi @mabenedi,

Any updates regarding the expected release of a feature?

**Comment by zakarel** [comment]
Commenting for updates

**Comment by asifkd012020** [comment]
Any updates on this feature?

**Comment by ericyew** [comment]
is this still being worked on?

**Comment by ericyew** [comment]
@mabenedi is this still being worked on?

**Comment by mabenedi** [comment]
@ericyew  Yes, still being worked on. Unfortunately still can't give a good ETA.

**Comment by ErezArbell** [comment]
We also need ACR to mirror and cache on-prem (on Azure) JFrog images registry

**Comment by zakarel** [comment]
hi @mabenedi , any new estimation for this feature. I have major customer still waiting and inquiring about it. 
Thanks

**Comment by mabenedi** [comment]
@zakarel Hi, it's still planned but unfortunately I still don't have a good ETA for you.

**Comment by m4rii0** [comment]
Hello @mabendi,

it's been roughly 3 months since the last update, could you please share any details about the development?

Thanks!

**Comment by brian-pickens** [comment]
Upvoting

**Comment by mabenedi** [comment]
@m4rii0 I'm not longer on the team. @JXavierMSFT should be able to give a status update

**Comment by brian-pickens** [comment]
@JXavierMSFT do you have any updates on this request?

**Comment by JXavierMSFT** [comment]
Hello @brian-pickens, 

My apologies for the delay in delivery of this feature. We are currently working on perfecting Docker Hub caching. However, once we finish perfecting Docker Hub caching. JFROG and ACR as an upstream is next on the list. Therefore, at the moment I don't have a solid delivery date I can provide. However, the moment we pick up this work again I will update this thread.

**Comment by rickardk-se** [comment]
@JXavierMSFT

While I appreciate the focus on quality, I'm concerned that striving for a 'perfect' solution for Docker Hub might delay the implementation of support for JFrog. A functional and reliable solution may be sufficient, as perfection can often be a moving target.

**Comment by diegolruiz** [comment]
Any updates regarding the expected release of a feature?

**Comment by JXavierMSFT** [comment]
Hey @rickardk-se and @diegolruiz, 

My apologies for the frustration while you are waiting on this feature. This is something we are working on. I don't have a timeline for you at this time. However, the second I have a solid delivery date. I will update this thread.

**Comment by yurymkomarov** [comment]
@JXavierMSFT is there a link to roadmap to track that feature delivery?

**Comment by m4rii0** [comment]
Hey there! @terencet-dev and @toddysm

We’d really appreciate some updates on how the development of this feature is going. Do you have an estimated ETA? It’s been about two years since the first timeline was shared, and right now we don’t have any visibility. Any insights would be super helpful!

**Comment by dash-mhp** [comment]
2 years - guess it will be an awesome feature indeed...
I mean, M$oft has to add another item in the dropdown-menu... that's like at least 3 months of work...
Then the actual function - 3 lines - so another 3 months...

**Comment by klbrg** [comment]
Getting something out (MVP) versus getting something perfect...

**Comment by wolszakp** [comment]
Any updates here?
